src/index.ts

```diff
@@ -13,7 +13,6 @@ export { FormattedDate } from 'FormattedDate/FormattedDate';
 export { default as FormField } from 'FormField';
 export { default as FormFieldError } from 'FormFieldError';
 export { default as FormFieldHint } from 'FormFieldHint';
-export { FileDownload } from 'FileDownload/FileDownload';
```

Comment:
Is already declared at https://github.com/wellcometrust/corporate-components/pull/269/files#diff-f41e9d04a45c83f3b6f6e630f10117feR10